travvy/src/components/Profile.vue

```diff
@@ -28,12 +29,18 @@
         <button class="button" @click="updateUserProfile()"><span>Update User Profile</span></button>
         <br>
         <br>
-
+        
         <button class="button" @click="signout"><span>Sign Out</span></button>
         <br>
         <br>
         <!-- Button to go to the home page --> 
         <router-link to="/Home" tag="button" class="button"><span>Return Home</span></router-link>
+        <br>
+        <br>
+        <button class="button"><span><a class="documents" href="https://firebasestorage.googleapis.com/v0/b/msci342project.appspot.com/o/test-ticket.png?alt=media&token=d730e012-57ca-4dd4-a870-a18002fbacbf" target="_blank">View Travel Documents</a></span></button>
```

Comment:
Oh amazing, this is where the user documents would be stored, correct? So excited about this new feature!